src/js/events/ChronosActions.js

```diff
@@ -46,10 +55,105 @@ const ChronosActions = {
     },
     {delayAfterCount: Config.delayAfterErrorCount}
   ),
+
+  fetchJobDetail: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      data: [
+        {name: 'embed', value: 'activeRuns'},
+        {name: 'embed', value: 'schedules'}
+      ],
+      success: function (response) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_SUCCESS,
+          data: response,
+          jobID
+        });
+      },
+      error: function (xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          jobID,
+          xhr
+        });
+      },
+      hangingRequestCallback: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ONGOING,
+          jobID
+        });
+      }
+    });
+  },
+
+  deleteJob: function (jobID) {
```

Comment:
We need a way to pass the `stopCurrentJobRuns` flag, to enforce the deletion of jobs.